Makefile

```diff
@@ -53,11 +60,11 @@ vet: ## Run go vet against code.
 	go vet ./...
 
 .PHONY: test
-test: manifests generate fmt vet tidy kustomize envtest ## Run tests.
-	rm -rf config/test && mkdir -p config/test/crd
-	$(KUSTOMIZE) build config/crd > config/test/crd/flinkoperator.k8s.io_flinkclusters.yaml
-	KUBEBUILDER_ASSETS="$(shell $(ENVTEST) use $(ENVTEST_K8S_VERSION) -p path)" go test ./... -coverprofile cover.out
```

Comment:
replacing only KUBEBUILDER_ASSETS wasn't working? the way it was should give you the same outcome

Reply:
It wasn't for some reason :|

Reply:
I cleared local cache of `bin` and the old way seems to work as well 👍